src/Filename/DefaultFilenameStrategy.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MakinaCorpus\DbToolsBundle\Filename;
```

Comment:
May be we should add an  `MakinaCorpus\DbToolsBundle\Storage` namespace and put the `DbToolsStorage` class in it an rename it `Storage` + these `FilenameStrategy` classes?